src/common/openssl/SSLCTXWrapper.cpp

```diff
@@ -179,12 +179,14 @@ bool SSL_CTX_wrapper::loadCert(
     } else if (!privateKeyFile.empty())
 	    THROWEXCEPTION("It makes no sense specifying a private key file without "
 		    "specifying a file that contains the corresponding certificate.");
+#ifdef DEBUG
```

Comment:
Can you briefly explain why we need this DEBUG switch here?

Reply:
Sure. The DPRINTF has functionality only when DEBUG is defined. For this reason, that entire if does nothing when DEBUG is not defined, and gcc complains about empty statements on the then and else branches.